record-and-playback/core/lib/recordandplayback/workers/events_worker.rb

```
@@ -42,6 +42,11 @@ def store_etherpad_events(target_dir)
         events = Nokogiri::XML(File.open("#{target_dir}/events.xml"))
         notes_id = BigBlueButton::Events.get_notes_id(events)
 
+        if notes_id.nil? || notes_id.empty?
+          @logger.warn("No notes_id found for #{@meeting_id}, skipping etherpad events")
+          return
+        end
+        
```

RuboCop (1.82.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)